src/ert/ensemble_evaluator/evaluator_tracker.py

```diff
@@ -64,11 +61,11 @@ def __init__(
 
     def _drain_monitor(self) -> None:
         asyncio.set_event_loop(asyncio.new_event_loop())
-        drainer_logger = logging.getLogger("ert.shared.ensemble_evaluator.drainer")
+        drainer_logger = logging.getLogger("ert.ensemble_evaluator.drainer")
         while not self._model.isFinished():
             try:
                 drainer_logger.debug("connecting to new monitor...")
-                with create_ee_monitor(self._ee_con_info) as monitor:
+                with Monitor(self._ee_con_info) as monitor:
```

Comment:
`create_ee_monitor` was a function which only called `Monitor.__init__`. I am not sure why this was introduced in the first place, it might have had a meaning at some point, or it was done to simplify mocking. I fixed the mocking so that it uses `Monitor.__init__` and removed this indirection.